projects/grange/src/lib/views/login.ts

```diff
@@ -1,60 +1,78 @@
 import { Component, OnInit, ChangeDetectionStrategy, ChangeDetectorRef } from '@angular/core';
 import { Grange } from '../grange.service';
 import { LoginInfo } from 'grange-core';
+import { ConfigurationService } from 'grange-core';
 import { markForCheck } from 'develop/pastanaga-angular/projects/pastanaga/src/lib/common/utils';
 
 @Component({
-    selector: 'grange-login',
-    templateUrl: './login.html',
-    styleUrls: ['./login.scss'],
-    changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'grange-login',
+  templateUrl: './login.html',
+  styleUrls: ['./login.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class LoginView implements OnInit {
-    useToken = false;
-    error = '';
-    isLogged: boolean;
-    allowRegister: boolean;
-    allowSocial: boolean;
-    allowForgot: boolean;
-    allowToken: boolean;
+  useToken = false;
+  error = '';
+  isLogged: boolean;
+  title: string;
+  allowRegister: boolean;
```

Comment:
all booleans should be assigne to `false` by default (unless `undefined` is a possible meaningful state)